docs/docs/writing-plugins/the-rules-api/testing-plugins.mdx

```diff
@@ -535,6 +535,33 @@ def test_build_ignore_dependency() -> None:
 
 `run_pants()` is hermetic by default, meaning that it will not read your `pants.toml`. As a result, you often need to include the option `--backend-packages` in the arguments to `run_pants()`. You can alternatively set the argument `hermetic=False`, although we discourage this.
 
+For example:
+
+```python
+def test_getting_list_of_files_from_a_target() -> None:
```

Comment:
would prefer to repeat the `from pants.testutil...` import line in each example to make them stand on their own.

Reply:
Great idea, done!